docs/howto/overriding-shell.txt

```diff
@@ -0,0 +1,47 @@
+=========================
+How to override the shell
+=========================
+
+Django provides functionality to auto-import project models in the shell. In your project, you might want to either avoid importing models or include additional imports in the shell. You can do this by subclassing the :djadmin:`shell` command and overriding a method.
+
+Setting up the app directory
```

Comment:
I think we can drop this section and just have a one-sentence link to `howto/custom-management-commands`. Developers doing shell customization presumably have some level of experience and don’t need everything spelled out.